api/v1beta1/types.go

```diff
@@ -324,6 +324,44 @@ type Image struct {
 	// Marketplace specifies an image to use from the Azure Marketplace
 	// +optional
 	Marketplace *AzureMarketplaceImage `json:"marketplace,omitempty"`
+
+	// CommunityGallery specifies an image to use from the Azure Community Gallery
+	// +optional
+	CommunityGallery *AzureCommunityGalleryImage `json:"communityGallery,omitempty"`
+}
+
+// AzureCommunityGalleryImage defines an image in the Azure Community Gallery ot use for VM creation.
```

Comment:
```suggestion
// AzureCommunityGalleryImage defines an image in the Azure Community Gallery to use for VM creation.
```

Reply:
Good reminder to fix typos in entire repo! :smile: Thanks for catching this and for reviewing!